x-pack/plugins/data_enhanced/server/search/session/get_search_status.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ApiResponse } from '@elastic/elasticsearch';
+import { ElasticsearchClient } from 'src/core/server';
+import { isAsyncSearchStatusResponse, SearchStatus } from './types';
+
+export async function getSearchStatus(client: ElasticsearchClient, asyncId: string) {
+  try {
+    const path = encodeURI(`/_async_search/status/${asyncId}`);
```

Comment:
Hmm, this is really going to have to look up the search strategy instead of being hardcoded here. Do you think we should implement an additional method on the strategy, like `getStatus` or something? Or should we just use the existing `search` method?

Reply:
@mayya-sharipova does this API return the status of EQL async searches as well?
If not, we'll need to figure something out. :)

Reply:
Yep we'll have to figure it out. Currently the status API is missing in EQL but I'd expect that we at least follow the same pattern (something like `_eql/status/${asyncId}`).

Reply:
This does mean that we won't support EQL in a search session for now.
It's too bad now though, because the apps in the first stage don't support EQL anyway (Dashboard, discover, lens).